### PR TITLE
Exclude the classloader for jasperreports-plugin

### PR DESCRIPTION
**Pull request by Postremus**
Basically the same fix as in #3079. Just new coordinates for the maven plugin.
Was reported over at [Jasper-report-maven-plugin/issues/52](https://github.com/pro-crafting/Jasper-report-maven-plugin/issues/52)

### CODE REVIEW

**Comment by mhagnumdw** [comment]
I tested it and it worked correctly.

Commands to generate the jar:
```
git clone git@github.com:projectlombok/lombok.git
cd lombok
git fetch origin pull/3568/head:pr-jasper
git switch pr-jasper
ant dist
```

In `eclipse.ini` I set `-javaagent:/tmp/lombok/dist/lombok-1.18.31.jar` and `jasperreports-plugin` worked as expected.

Any date to accept this PR? TKs!

**Comment by Postremus** [comment]
@Rawi01 I updated the authors and changelog file.

File: AUTHORS [hunk 1]
```diff
@@ -56,5 +56,6 @@ Till Brychcy <till.brychcy@mercateo.com>
 Victor Williams Stafusa da Silva <victorwssilva@gmail.com>
 Yonatan Sherwin <yonatansherwin@gmail.com>
 Yun Zhi Lin <yun@yunspace.com>
+Martin Panzer <postremus1996@googlemail.com>
```
Comment (Rawi01) [inline AUTHORS:59]: Can you move yourself to the correct line? The list is ordered by name.

**Review by Rawi01 — APPROVED** [review]

File: AUTHORS (hunk 1 above)
Comment (Postremus) [reply AUTHORS]: done

**Comment by Postremus** [comment]
Any chance to get a new release with this bug fix soon(ish)?

**Comment by mhagnumdw** [comment]
It hasn't even come out in the edge version yet. Any predictions for release?

**Pull request merged**